bedrock/firefox/templates/firefox/includes/download-button.html

```diff
@@ -4,7 +4,7 @@
 
 {% macro alt_buttons(builds) %}
   <div class="download download-dumb">
-    <h4>{{ _('Download Firefox') }} — {{ locale_name|safe }}</h4>
+    <p role="heading" class="download-heading"><b>{{ _('Download Firefox') }} — {{ locale_name|safe }}</b></p>
```

Comment:
I guess this also brings up the question of whether we should add the attribute `aria-labeledby` to the `ul`. That means the `p` will need an id attribute as well. As the emphasis is not visual but more for auditory purposes, I would think the mapping for `role="heading"` would suffice and that you are correct, we probably do not need either `strong` nor `b`